spring-integration-redis/src/test/java/org/springframework/integration/redis/inbound/RedisQueueMessageDrivenEndpointTests.java

```diff
@@ -78,24 +81,38 @@
 @DirtiesContext
 public class RedisQueueMessageDrivenEndpointTests extends RedisAvailableTests {
 
+	public static final String TEST_QUEUE = "testQueue";
```

Comment:
Don't we need to randomize this to avoid collisions?

Reply:
Yeah... I thought about that, but looks like the problem is more about not cleaned key in Redis.
If you insist, I can do that now or let's deffer when we fail again! 😄

Reply:
No; it's fine.